### ISSUE TITLE
CST: Hubless spoke page and IA needs review

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#41697]
# Hubless spoke pages needs review

- [Your compensation claim pages](https://staging.va.gov/track-claims/your-claims/600219085/status?postLogin=true)
- Profile pages
- [Mural outline of the issue](https://app.mural.co/t/vsa8243/m/vsa8243/1653404244021/d2fdb4d36539e09705429286accaf119db765664?sender=u2134c22982ad9c7b41798011)

Hubless spoke pages are pages that together as a group form one "hub" of information. The group itself _is_ the abstract hub, which creates a number of IA and a11y issues including, but not limited to:

<hr/>

## IA issues
<img width="1127" alt="Screen Shot 2022-11-08 at 9 19 17 AM" src="https://user-images.githubusercontent.com/14154792/200588949-31697c25-1ece-4c21-a53b-3775eb64686e.png">

Possible ways forward:
- [ ] conduct heuristic evaluation or content/IA discovery (could we plug in someone like @Evafloyd to help?)
- [ ] conduct technical discovery to determine which recommendations from the prior step are viable

<hr/>

### Duplicative titles `defect-3`
These tabs feature different claims but share the same title

<img width="730" alt="Screen Shot 2022-11-08 at 9 30 38 AM" src="https://user-images.githubusercontent.com/14154792/200591449-aad20504-6116-401e-a1e8-b24b1659e04b.png">

- [x] #60900

<hr/>

### Generic `h1` `resolved`
Every unique claim shares the same generic h1 making it more difficult for screen reader users to quickly determine if they’ve clicked on the right claim. These are all separate claims. To a screen reader user who determines where they've navigated to by jumping to the `h1`, it's unclear immediately which claim they're actually viewing.

<img width="1083" alt="Screen Shot 2022-11-08 at 9 31 47 AM" src="https://user-images.githubusercontent.com/14154792/200591734-661885ea-8d19-4ff4-941e-bc1156cf31cf.png">

- [x] Resolved using focus management documented in [Address Accessibility Issues in tabs on Clams Status Details](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47242)
- [ ] ⚠️ needs ticket if redesigned

<hr/>

### `h2` should not be a `h2` `defect-3`
"What you've claimed: [ ]" reads more like descriptive text to the `h1` and doesn't start a separate section, so it shouldn't be a `h2`. As a result, if the page's headings were translated into a table of contents or outline, it would read like this:

- Your compensation claim
  - What you've claimed: arthritis (new)
    - Status
      - Claim received
      - etc etc

<img width="305" alt="Screen Shot 2022-11-08 at 9 32 47 AM" src="https://user-images.githubusercontent.com/14154792/200592012-78faf494-f589-4375-8223-cc6d9a8e4158.png">

Compare that to an outline like this where what its claimed simply exists as content that's part of the section describing the compensation claim

- Compensation claim filed on December 14, 2020
  - Status 
    - Claim received

<img width="384" alt="Screen Shot 2022-11-08 at 9 37 28 AM" src="https://user-images.githubusercontent.com/14154792/200593134-24464782-441d-46fd-b1f7-89df8cec4565.png">

- [ ] ⚠️ needs ticket

<hr/>

### "View details" goes to the status page, not the details page

<img width="773" alt="Screen Shot 2022-11-08 at 9 37 57 AM" src="https://user-images.githubusercontent.com/14154792/200593233-610f028e-e668-4784-af0a-ebf0a1e82acb.png">

A semantics mismatch exists on the list of claims as clicking on "view details" takes the user to the "status" tab (where it is in the process) instead of the "details" tab (information about the claim).
- [ ] ⚠️ needs ticket

<hr/> 

## Dishonest tabs
Because these are links styled to look like tab buttons, keyboard users may become frustrated that they can’t navigate between tabs using their left and arrow keys (which is the expected wai-aria pattern)
- [ ] [508-defect-2 [KEYBOARD, COGNITION] Tablist on compensation claims are links styled as a tablist without keyboard functionality](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41449)
- [ ] [Address Accessibility Issues in tabs on Clams Status Details](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47242)

<hr/>

## Hidden h3
There’s a hidden h3 on this page to represent “status” which (a) is not visible to sighted screen reader users and (b) is already significantly deep in the headings hierarchy despite it being the main content of the current page
- [ ] [Address Accessibility Issues in tabs on Clams Status Details](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47242)

## Other heading issues

<img width="1791" alt="Screenshot 2023-06-12 at 4 04 49 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/9f709665-ba9d-4abe-9c4a-56bce45aa10a">
<img width="1282" alt="Screenshot 2023-06-12 at 4 10 32 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/d22a1903-ec38-403a-b277-5312cb4db678">

On a burials claim:
- the hidden `h3` for status is missing
- alert card is missing a heading
- "Next steps" skips a heading 
- "Payments" skips a heading

Upon redesigning the pages, we should create acceptance criteria for logically ordered headings

**Comment by joshkimux** [comment]
@jacobworrell , some updates here as promised ❇️ 
- I've removed this as a `defect-2`
- I've split this wicked ticket apart into smaller issues to be addressed
  - The larger IA issues need a little love. I left ideas on how we can push forward there.
  - The other issues related to keyboard navigation are in the works ✨ I've attached the tickets that will resolve them here.

Let me know if there's anything else I can do to help push this forward. Thank you again for all your support!

**Comment by joshkimux** [comment]
Added heading issues

**Comment by skylerschain** [comment]
I think we mostly fixed these issues with V2. 

- Unique H1s by including the "Submitted on" subtitle for each claim
- "What you've claimed" is now a true H2 with a `<ul>` below
- Tabs now have unique titles
- Details page navigation resolved bc we no longer have "Details" tab
- Links that look like buttons was resolved with the new tab design

@davidakennedy Can you confirm if there's anything I'm missing, or can we close this ticket out?

**Comment by davidakennedy** [comment]
@skylerschain The only thing we didn't do here in what Josh calls out is the removal of the h2, called out as "h2 should not be a h2 defect-3".

We still have a stacked heading structure: A h1 followed immediately by a h2. Example:

```html
<h1>Your compensation claim
Received on September 10, 2016</h1>
<h2>What you’ve claimed</h2>
```

That's not ideal, but it's not a huge problem either. I tend to err on the side of more headings, so I think we can leave it unless we hear feedback otherwise.

So I'm good to close this.
